### DISCUSSION TITLE
Columnar file view mode (as in the mac Finder)

### DISCUSSION THREAD

**Discussion by ragunyrasta** [elementary/files#2349] [Q&A] [answered]
The Mac Finder has a Columnar view which enables viewing a large amount of information at the same time (files, folders, subfolders, subsubfolders and so on.) A picture is attached. What would it take to make this happen in Elementary's Files app?

<img width="858" alt="image" src="https://github.com/elementary/files/assets/22959128/4cab2fef-5db5-4576-8719-9206c02de0f1">

**Comment by danirabbit** [comment, marked as answer]
You can select "View in columns" from the view mode switcher in the top left corner of the window or with the keyboard shortcut <kbd>Ctrl</kbd>+<kbd>3</kbd>

![Screenshot from 2023-10-25 08 33 09](https://github.com/elementary/files/assets/7277719/096e7422-14f5-4987-a9e7-d7cca44caceb)

**Comment by ragunyrasta** [comment]
Cool! Now I’ll have to seriously consider migrating from Mac to Elementary

Is it possible to make the view compact, so we can see a lot more items in the same area?

**Reply by danirabbit** [reply]
There are zoom controls in the gear menu in the top right!

**Reply by jeremypw** [reply]
Or use `<Ctrl>Scroll` or `<Ctrl>plus`/`<Ctrl>minus`
